### ISSUE TITLE
Mutli GPU Training 

### ISSUE THREAD

**Issue by Venky0892** [ultralytics/yolov5#7608]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hi, I trying to train around 200K images via Tesla V100, I have 4 of these in my compute instance. I'm training with this command "python -m torch.distributed.launch --nproc_per_node 4" as mentioned in the documentation. But I could only able see just one GPU is active, I'm not sure why? I checked the GPU usage metric in Wandb tool. Can someone help me with this? If I run normal training without distributed launch I could see all the GPU are being used.

**Comment by glenn-jocher** [comment]
@Venky0892 see **Multi-GPU Training** tutorial for correct commands:

## YOLOv5 Tutorials

* [Train Custom Data](https://docs.ultralytics.com/yolov5/tutorials/train_custom_data)&nbsp; 🚀 RECOMMENDED
* [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/)&nbsp; ☘️  RECOMMENDED
* [Weights & Biases Logging](https://github.com/ultralytics/yolov5/issues/1289)&nbsp; 🌟 NEW
* [Roboflow for Datasets, Labeling, and Active Learning](https://docs.ultralytics.com/yolov5/tutorials/roboflow_datasets_integration)&nbsp; 🌟 NEW
* [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training)
* [PyTorch Hub](https://docs.ultralytics.com/yolov5/tutorials/pytorch_hub_model_loading)&nbsp; ⭐ NEW
* [TFLite, ONNX, CoreML, TensorRT Export](https://docs.ultralytics.com/yolov5/tutorials/model_export) 🚀
* [Test-Time Augmentation (TTA)](https://docs.ultralytics.com/yolov5/tutorials/test_time_augmentation)
* [Model Ensembling](https://docs.ultralytics.com/yolov5/tutorials/model_ensembling)
* [Model Pruning/Sparsity](https://docs.ultralytics.com/yolov5/tutorials/model_pruning_and_sparsity)
* [Hyperparameter Evolution](https://docs.ultralytics.com/yolov5/tutorials/hyperparameter_evolution)
* [Transfer Learning with Frozen Layers](https://docs.ultralytics.com/yolov5/tutorials/transfer_learning_with_frozen_layers)&nbsp; ⭐ NEW
* [Architecture Summary](https://docs.ultralytics.com/yolov5/tutorials/architecture_description)&nbsp; ⭐ NEW

Good luck 🍀 and let us know if you have any other questions!

**Comment by Venky0892** [comment]
Hi @glenn-jocher 
I referred to the documentation for Multi GPU Training:
Here the Command : "$ python -m torch.distributed.launch --nproc_per_node 4 train.py --batch 128 --data furniture.yaml --weights yolov5s.pt --device 0,1,2,3

I want to use four GPU's so gave --nproc_ per_node = 4

<img width="724" alt="Screenshot 2022-04-28 at 6 09 25 PM" src="https://user-images.githubusercontent.com/49713881/165796914-3dd4ae91-a475-4b05-808a-08a2192e8019.png">

I could only see GPU 0 is active and running. I'm not sure it could be great if you can assits me in this

**Comment by glenn-jocher** [comment]
@Venky0892 The command you provided looks correct, so it's strange that only GPU 0 is active. Would you mind checking if the `nvidia-smi` command correctly shows all 4 GPUs under normal circumstances? Additionally, could you please verify that your system meets the requirements outlined in the [Multi-GPU Training](https://docs.ultralytics.com/yolov5/tutorials/multi_gpu_training) tutorial in our documentation? Thank you!
